docs/developer/ios-sdk.md

```diff
@@ -153,78 +153,5 @@ NSMutableDictionary *messengerData = [[NSMutableDictionary alloc] init];
         // do your stuff
 }];
 ```
-<!-- 
```

Comment:
why deleted?

Reply:
Some pages of doc do not shown subtitles in right column because of additional comments. That's why deleted comments.